src/analytics/namespaces/dashboard.py

```diff
@@ -337,16 +322,9 @@ def post(self):
         )
 
 
-@rest_api_v2.route("/dashboard/exceedances-devices")
-class ExceedancesResource2(Resource):
+@dashboard_api.route("/exceedances-devices")
+class DeviceExceedancesResource(Resource):
```

Comment:
Hi @Mnoble-19 , thanks for renaming the function and class names accordingly. This is brilliant, please keep it up. We need to move to a future where we use proper function and variable names thus almost eliminating the need for using comments in code.